bedrock/firefox/templates/firefox/welcome/page16.html

```diff
@@ -41,6 +72,13 @@ <h1 class="c-main-title">This Cybersecurity Month, take 20% off your first year<
           link_text='Get Mozilla VPN',
           class_name='mzp-t-product mzp-t-xl',
           page_anchor="#pricing",
+          optional_parameters={
+            'utm-medium': _entrypoint,
```

Comment:
As long as we’re running old GA we should not be attaching utm params to same site links like this. Doing so will inflate the number of sessions we see, as clicking the link will start a new session.